torch/nn/utils/_stateless.py

```diff
@@ -1,29 +1,62 @@
 import contextlib
 
-import torch
+import torch.jit
+
+
+def _change_class(module) -> None:
+    cls = module.__class__
+    func_params = module._functional_parameters
+
+    def _getattribute(self, name):
+        if name in func_params:
+            return func_params[name]
+        return object.__getattribute__(self, name)
```

Comment:
`super().__getattribute__` here right?

Reply:
that leads to infinite recursion :D, so we need to use the base object method.

Reply:
tried again to fix it and it worked now, seems I was messing with something else originally :D